index.js

```diff
@@ -205,7 +205,7 @@ module.exports = {
     'no-lone-blocks':        [ 'error' ], // Stylistic.
     'no-lonely-if':          [ 'error' ], // Stylistic.
     'no-loop-func':          [ 'error' ], // Best practices.
-    'no-magic-numbers':      [ 'error' ], // Best practices.
+    'no-magic-numbers':      null,
```

Comment:
Looks like 0 was correct
```suggestion
    'no-magic-numbers':      0,
```